src/nft/components/explore/Banner.tsx

```diff
@@ -112,74 +112,64 @@ const CollectionDetails = ({
   rank: number
   hovered: boolean
 }) => (
-  <Column className={styles.collectionDetails} paddingTop="24">
-    <div className={styles.volumeRank}>#{rank} volume in 24hr</div>
-    <Row>
-      <Box as="span" marginTop="16" className={clsx(header1, styles.collectionName)}>
-        {collection.name}
-      </Box>
-      {collection.isVerified && (
-        <Box as="span" marginTop="24">
-          <VerifiedIcon height="32" width="32" />
-        </Box>
-      )}
-    </Row>
-    <Row className={bodySmall} marginTop="12" color="explicitWhite">
-      <Box>
-        <Box as="span" color="darkGray" marginRight="4">
-          Floor:
+  <Box
+    as="section"
+    className={section}
+    display="flex"
```

Comment:
Why aren't these defined in the css for `section`?

Reply:
`section` was previously used in other parts of `/nfts` and needed to be kept simple. Now its only used here so I moved it out and combined with these styles.